libfreerdp/core/pkcs11/cert_info.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * PKCS #11 PAM Login Module
```

Comment:
is this module not available as a library? maybe better to use as external dependency than to copy the code